### PR TITLE
Caching updates

### PR DESCRIPTION
**Pull request by loosebazooka**
merging of @thesayyn #1242 into main

### CODE REVIEW

**Comment by loosebazooka** [comment]
before merge: change cloud build to look for build file in .cloudbuild/cloudbuild.yaml

File: .cloudbuild/release.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+echo '717a44dd526c574b0a0edda1159f5795cc1b2257db1d519280a3d7a9c5addde5 bazel-remote' | sha256sum --check
+chmod +x bazel-remote
+mkdir .logs
+./bazel-remote --max_size 8 --dir ~/.cache/bazel-remote --experimental_remote_asset_api --grpc_address 0.0.0.0:4700 --gcs_proxy.bucket $REMOTE_CACHE_GCS --gcs_proxy.use_default_credentials > .logs/bazel-remote.log 2>&1 &
+
+# install bazel 6
+apt-get install bazel-6.0.0
+ln -sf /usr/bin/bazel-6.0.0 /usr/bin/bazel
+
+# setup docker-credential-gcr
```
Comment (thesayyn) [inline .cloudbuild/release.sh:15]: nit: turned out to be unnecessary. can be removed

**Review by thesayyn — APPROVED** [review]

File: .cloudbuild/release.sh (hunk 1 above)
Comment (loosebazooka) [reply .cloudbuild/release.sh:15]: nice I can use that to retrigger the GHAs.

**Review by bobcallaway — APPROVED** [review]

**Comment by loosebazooka** [comment]
Once again. Do not merge till the cloud build trigger is updated

**Pull request merged**